### PR TITLE
connected api using axios

### PR DESCRIPTION
**Pull request by emanuelLeiva**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -2,30 +2,30 @@ import 'react-native-gesture-handler';
 import React from 'react';
 import {ThemeProvider} from 'styled-components/native';
 import theme from './src/themes';
-import crypto from './src/screens/Cryptos';
+import crypto from './src/screens/cryptos';
```
Comment (tomasdematos) [inline App.tsx:5]: it should be 

import Cryptos from './src/screens/Cryptos';

File: App.tsx [hunk 2]
```diff
@@ -2,30 +2,30 @@ import 'react-native-gesture-handler';
...
 import {NavigationContainer} from '@react-navigation/native';
 import {createStackNavigator} from '@react-navigation/stack';
-import addCrypto from './src/screens/Add';
+import addCrypto from './src/screens/add';
```
Comment (tomasdematos) [inline App.tsx:8]: it should be
import AddCrypto from './src/screens/AddCrypto';

File: src/screens/cryptos/index.tsx [hunk 3]
```diff
@@ -1,49 +1,34 @@
...
+
+function List() {
+  const [cryptos, setCryptos] = useState<Crypto[]>([]);
+
+  useEffect(() => {
+    setCryptos([]);
+    const getCryptos = async (coin: string) => {
+      const URL = `https://data.messari.io/api/v1/assets/${coin}/metrics#`;
+      const cryptos = await axios.get(URL);
+      console.log('Cryptos', cryptos.data.data);
```
Comment (tomasdematos) [inline src/screens/cryptos/index.tsx:20]: delete this console.log

File: src/screens/cryptos/index.tsx [hunk 4]
```diff
@@ -1,49 +1,34 @@
...
+      setCryptos(prev => [...prev, cryptos.data.data]);
+      console.log(cryptos);
```
Comment (tomasdematos) [inline src/screens/cryptos/index.tsx:22]: delete the log

File: src/screens/cryptos/index.tsx [hunk 5]
```diff
@@ -1,49 +1,33 @@
...
 
-const List = () => (
+function List() {
+  const [cryptos, setCryptos] = useState<Crypto[]>([]);
+
+  useEffect(() => {
+    setCryptos([]);
+    const getCryptos = async (coin: string) => {
+      const URL = `${API_URL}${coin}/metrics#`;
+      console.log('API_URL', API_URL);
```
Comment (VicenBayugar) [inline src/screens/cryptos/index.tsx:19]: delete this log

**Pull request merged**